core/quick-silver/quick-silver.ts

```
@@ -0,0 +1,196 @@
+  AllDocsQueryOpts,
+  AllDocsResponse,
+  ListenerFn,
+  MapFn,
+  IndexKeyType,
+  DocFragment,
+  QueryOpts,
+  QueryResult,
+  Attachable,
+  Attached,
+} from "@fireproof/core-types-base";
+import { BuildURI, Lazy, Logger, OnFunc, URI } from "@adviser/cement";
```

ESLint

[error] 'URI' is defined but never used. Allowed unused vars must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] '@fireproof/core-types-base' imported multiple times.

(import/no-duplicates)